defaults/python/lib/moonlightproxy.py

```diff
@@ -87,12 +89,20 @@ async def start(self, hostname: str, host_app: str, audio: Optional[str] = None,
                 args += ["--hdr" if resolution["hdr"] else "--no-hdr"]
             if resolution["bitrate"]:
                 args += ["--bitrate", f"{resolution['bitrate']}"]
+            if resolution["video_codec"] and resolution["video_codec"] != "Default":
```

Comment:
```suggestion
            if resolution["video_codec"]:
```
`None` should cover the `Default` option.